app/generator/content/summary/get-sbi.js

```diff
@@ -0,0 +1,12 @@
+const getSBI = (sbi) => {
+  return {
+    columns: [
+      { width: 200, text: 'Single business identifier (SBI):' },
+      { width: '*', text: sbi }
+    ],
+    style: 'column',
+    columnGap: 10
+  }
+}
+
+module.exports = getSBI
```

Comment:
It looks like `sbi` and `businessName` column will always have the same formatting style, we may consider creating column templates for similar columns.